tests/check_consistent.py

```diff
@@ -83,6 +83,19 @@ def check_stubs() -> None:
             assert not py_files_present, error_message
 
 
+def check_distutils() -> None:
+    """Check whether all setuptools._distutils files are re-exported from distutils."""
+
+    def all_relative_paths_in_directory(path: Path) -> set[Path]:
+        return {pyi.relative_to(path) for pyi in path.rglob("*.pyi")}
+
+    all_setuptools_files = all_relative_paths_in_directory(Path("stubs", "setuptools", "setuptools", "_distutils"))
+    all_distutils_files = all_relative_paths_in_directory(Path("stubs", "setuptools", "distutils"))
+    extra_files = all_setuptools_files - all_distutils_files
```

Comment:
Optional: we could do this as a meta-test to ensure that this test doesn't become irrelevant, e.g. if we change our directory structure or something in the future:

```suggestion
    assert all_setuptools_files and all_distutils_files, "Looks like this test might be out of date!"
    extra_files = all_setuptools_files - all_distutils_files
```